test/parallel/test-assert.js

```diff
@@ -716,7 +716,7 @@ common.expectsError(
   () => {
     a(
       (() => 'string')()
-      // eslint-disable-next-line
+      // eslint-disable-next-line operator-linebreak
```

Comment:
CI is failing because the message check in L725-L736 needs to be updated to match.

Reply:
Also: Argh! Fixed.